wallet/res/drawable/splash_background.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android"
+    android:opacity="opaque">
+
+    <item
+        android:drawable="@drawable/splash_activity_background"
+        android:gravity="fill_vertical" />
+
+    <item
+        android:drawable="@drawable/splash_activity_slogan"
```

Comment:
This file appears to be missing.

Reply:
splash_activity_slogan

Reply:
https://github.com/HashEngineering/dash-wallet/pull/223/commits/ee30ba3f51a28471d113ea74d8b1b943e7953707

This commit from the #223 PR has the files